### PR TITLE
fixed incorrect description of coin spend

### PR DESCRIPTION
**Pull request by darkverbito**
Corrected the wording in the coin spend information to be consistent with the desired final state.

### CODE REVIEW

File: docs/04coin-set-model/intro.md [hunk 1]
```diff
@@ -55,4 +55,4 @@ Value may only be added to the coin set via the pre-farm (a one-time occurrence)
 Typically, in a block's combined spend bundle, value added will be equal to value spent, other than the block rewards. By definition, there are two possible exceptions:
 
 * Value added > value spent -- This is not allowed, so the transaction will be rejected. The rejection will usually happen at the mempool level, though a malicious actor could write their own mempool to accept the transaction, in which case the blockchain will reject it.
-* Value added < value spent -- This is allowed, so the transaction will succeed. In this case, the remaining value will be rewarded to the farmer of the block containing the transaction.
\ No newline at end of file
+* Value added =< value spent -- This is allowed, so the transaction will succeed. If the value added is less than the value spent the remaining value will be rewarded to the farmer of the block containing the transaction awarded as a tip (you probably don't want to do that!).
```
Comment (danieljperry) [inline docs/04coin-set-model/intro.md:58]: If you think the extra wording is needed for clarification, that's fine. However, I don't know what "=<" means.

File: docs/04coin-set-model/coin_set_vs_account.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ In Chia's coin set model, there are no accounts or balances. There are only coin
 
 A typical Chia transaction would look like the following:
 
-Alice wants to send 1 XCH to Bob. Alice has five coins in her wallet: four worth 0.2 XCH apiece, and one worth 0.7 XCH. Her wallet automatically selects two of the coins worth 0.2 XCH and the 0.7 XCH coin to be spent. All three coin spends happen simultaneously, along with two new coin creations: one worth 1 XCH which goes to Bob's wallet, and one worth 0.1 XCH which goes to Alice's wallet as "change." Alice now has two coins worth 0.2 XCH and one worth 0.1 XCH. Bob has one coin worth 1 XCH. The total value owned by Alice and Bob has not changed -- it was 1.5 XCH both before and after the transaction was processed.
```
[the remark this answers is not in the document]
Comment (darkverbito) [reply docs/04coin-set-model/coin_set_vs_account.md:27]: Then the table is wrong.

File: docs/04coin-set-model/intro.md (hunk 1 above)
Comment (darkverbito) [reply docs/04coin-set-model/intro.md:58]: Less than or equal to.

File: docs/04coin-set-model/coin_set_vs_account.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (danieljperry) [reply docs/04coin-set-model/coin_set_vs_account.md:27]: Ah, yes that's the problem. Alice has 3 coins after the transaction. One of them is her change coin. The XCH amounts are correct, though.

File: docs/04coin-set-model/intro.md (hunk 1 above)
Comment (danieljperry) [reply docs/04coin-set-model/intro.md:58]: If the intention was for it to mean "less than or equal to" then that's incorrect. If the two values are equal, then that's the typical case. The two bullet points are the exceptions, when the values are not equal.

**Comment by darkverbito** [comment]
Are you going to merge this or update the table?

**Comment by darkverbito** [comment]
Ok, table is updated with the original text restored.

**Review by danieljperry — APPROVED** [review]
Looks good.

**Comment by danieljperry** [comment]
@darkverbito All set, thanks for your contribution!

**Pull request merged**